doc/sphinx/source/community/release_strategy.rst

```diff
@@ -352,21 +359,7 @@ Announce the name of the branch in an issue and ask the members of the
 `ESMValTool development team <https://github.com/orgs/ESMValGroup/teams/esmvaltool-developmentteam>`__
 to run their favourite recipe using this branch.
 
-5. Cherry pick bugfixes into the release branch
```

Comment:
I noticed that there are still mentions of cherry-picking in other places in this document. Would it make sense to update those too?

Reply:
I noticed that you thought it was better to stick with the feature freeze and cherry-picking bug fixes into the release branch here: https://github.com/ESMValGroup/ESMValCore/pull/1689#discussion_r947650781, instead of allowing an endless stream of updates into the release candidate. I also think this is a good idea (that's why it is in the text as it is now), but this was done differently the past few releases.

@ESMValGroup/technical-lead-development-team Is this something we need to talk about at one of our upcoming meetings so we have a shared understanding of how we do releases?

Reply:
Yes, but that advice is contrary to having a feature freeze unless the feature freeze means that only bug fixes can be merged into the `main` branch while the release is in progress. Let's talk about this with the team next week or at the next tech lead meeting.